assets/i18n/en.json

```diff
@@ -764,6 +764,7 @@
     "I18N_TOPNAV_CONTRIBUTOR_DASHBOARD": "Contributor Dashboard",
     "I18N_TOPNAV_CREATOR_DASHBOARD": "Creator Dashboard",
     "I18N_TOPNAV_DONATE": "Donate",
+    "I18N_TOPNAV_DONATE_DESCRIPTION": "Your contributions help provide quality education to all.",
```

Comment:
We will need to add a descriptive text about it in qqq.json. This will fix the backend tests. Ensure key in en.json is at same place as that in qqq.json

Reply:
> We will need to add a descriptive text about it in qqq.json. This will fix the backend tests. Ensure key in en.json is at same place as that in qqq.json

Thanks @Rijuta-s ! Made this change and also updated the UI a bit as you had mentioned.